jetty-http/src/main/java/org/eclipse/jetty/http/UriCompliance.java

```diff
@@ -91,19 +130,23 @@ public static UriCompliance valueOf(String name)
     /**
      * Create compliance set from string.
      * <p>
-     * Format:
+     * Format: &lt;BASE&gt;[,[-]&lt;violation&gt;]...
      * </p>
+     * <p>BASE is one of </p>
      * <dl>
      * <dt>0</dt><dd>No {@link Violation}s</dd>
      * <dt>*</dt><dd>All {@link Violation}s</dd>
-     * <dt>RFC2616</dt><dd>The set of {@link Violation}s application to https://tools.ietf.org/html/rfc2616,
-     * but not https://tools.ietf.org/html/rfc7230</dd>
-     * <dt>RFC7230</dt><dd>The set of {@link Violation}s application to https://tools.ietf.org/html/rfc7230</dd>
-     * <dt>name</dt><dd>Any of the known modes defined in {@link UriCompliance#KNOWN_MODES}</dd>
+     * <dt>&lt;name&gt;</dt><dd>The name of a set of allowed violations from {@link UriCompliance#KNOWN_MODES}</dd>
      * </dl>
+     * </p>
```

Comment:
Remove this extra `</p>`.

Reply:
I can never remember if the list should be within the paragraph or not.... seems to format the same.... so OK.